lib/oneview-sdk/resource/api300/c7000/server_hardware.rb

```diff
@@ -75,6 +76,38 @@ def patch(operation, path, value = nil)
           @client.response_handler(response)
         end
 
+        # Add one scope to the enclosure
+        # @param [OneviewSDK::API300::C7000::Scope] scope The scope resource
+        # @raise [OneviewSDK::IncompleteResource] if the uri of scope is not set
+        def add_scope(scope)
+          scope.ensure_uri
+          patch('add', '/scopeUris/-', scope['uri'])
+        end
+
+        # Remove one scope from the enclosure
+        # @param [OneviewSDK::API300::C7000::Scope] scope The scope resource
+        # @return [Boolean] True if the scope was deleted and false if enclosure has not the scope
+        # @raise [OneviewSDK::IncompleteResource] if the uri of scope is not set
+        def remove_scope(scope)
+          scope.ensure_uri
+          scope_index = @data['scopeUris'].find_index { |uri| uri == scope['uri'] }
+          if scope_index
```

Comment:
> Somehow my comment was marked in a outdated change... anyway, here it is again:

It would be better to use a guard clause here:

```Ruby
return false unless scope_index
patch('remove', "/scopeUris/#{scope_index}", nil)
true
```